docs/units/equivalencies.rst

```diff
@@ -234,9 +230,53 @@ example converts the FWHM to sigma::
     >>> beam_sigma = beam_fwhm * fwhm_to_sigma
     >>> omega_B = 2 * np.pi * beam_sigma**2
     >>> freq = 5 * u.GHz
-    >>> u.Jy.to(u.K, equivalencies=u.brightness_temperature(omega_B, freq))  # doctest: +FLOAT_CMP
-    19.553928332631582
+    >>> (1*u.Jy/omega_B).to(u.K, equivalencies=u.brightness_temperature(freq))  # doctest: +FLOAT_CMP
+    <Quantity 19.553932298231704 K>
+
+You can also convert between ``Jy/beam`` and ``K`` by specifying the beam area::
+
+    >>> import numpy as np
+    >>> beam_fwhm = 50*u.arcsec
+    >>> fwhm_to_sigma = 1. / (8 * np.log(2))**0.5
+    >>> beam_sigma = beam_fwhm * fwhm_to_sigma
+    >>> omega_B = 2 * np.pi * beam_sigma**2
+    >>> freq = 5 * u.GHz
+    >>> (1*u.Jy/u.beam).to(u.K, u.brightness_temperature(freq, beam_area=omega_B))  # doctest: +FLOAT_CMP
+    <Quantity 19.553932298231704 K>
+
+Finally, there is an equivalency that allows you to convert from Jansky to Kelvin.
+In this case, the Jansky unit is *implicitly* Jansky/beam.  Because of the implicit
+assumed per beam unit, this approach is deprecated.::
```

Comment:
This needs to be removed, there is no need to promote deprecated usage in our docs.